src/main/java/com/griddynamics/cd/nrp/internal/uploading/impl/UploadEventListenerImpl.java

```diff
@@ -73,23 +73,23 @@ public void onArtifactUploading(RepositoryItemEventStore event) {
                 ArtifactMetaInfo metaInfo = new ArtifactMetaInfo(configurationsManager.getConfiguration().getMyUrl(), gav.getGroupId(), gav.getArtifactId(), gav.getVersion(), repo.getId());
                 metaInfo.setClassifier(gav.getClassifier());
                 metaInfo.setExtension(gav.getExtension());
-                ArtifactStatus artifactStatus = getArtifactStatus(metaInfo);
+                ArtifactStatus artifactStatus = null;
                 if (!gav.isSignature() && !gav.isHash()) {
```

Comment:
We agree with @kevstigneev to write tests for this method while implementing issue "Automated tests" after release 1.0